src/main/java/com/megastore/model/User.java

```diff
@@ -0,0 +1,36 @@
+package com.megastore.model;
+
+import com.megastore.type.RoleType;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+
+//@Table(name="userprofile", catalog = "public")
+@Getter
+@Setter
+@Entity
+@Table(name = "users")
+public class User extends BaseEntity {
+    @Column(nullable = false, unique = true)
+    private String email;
+    @Column(nullable = false)
+    private String password;
```

Comment:
The user will not have a password field because authentication will be done with auth0. This service on its side authorizes the user and only gives us his data